CactbotOverlay/FFXIVProcess.cs

```diff
@@ -512,6 +573,16 @@ public class EntityData {
             data.debug_job += string.Format("{0:x2}", job_bytes[i]);
           }
         }
+
+        for (int i = 0; i < data.statusEffects.Length; i++) {
+          var currentBase = kEntityStructureOffsetCharacterDetails + kEntityStructureOffsetStatusEffects + i * StatusEffect.structSize;
+
+          var id = BitConverter.ToUInt16(bytes, currentBase + StatusEffect.offsetId);
```

Comment:
Would it be possible to not include all of the blank status effects with id zero?